### PR TITLE
fix issue #392

### LINKED ISSUE #392: Patient minimum filters
If patients are less that 10 unique don't show patient list.

### PR DESCRIPTION
**Pull request by mingjiecn**
close #392 
image

### CODE REVIEW

**Comment by mingjiecn** [comment]
Just add period at the end of sentence.

File: src/encoded/static/components/search.js [hunk 1]
```diff
@@ -1107,12 +1109,18 @@ export class ResultTable extends React.Component {
                 />
                 {context.notification === 'Success' ?
                     <div className="search-results__result-list">
-                        <h4>Showing {results.length} of {total} {label}</h4>
-                        <SearchControls context={context} visualizeDisabledTitle={visualizeDisabledTitle} onFilter={this.onFilter} showResultsToggle />
-                        {!(actions && actions.length > 0) ?
-                            <CartSearchControls searchResults={context} />
-                        : null}
-                        <ResultTableList results={results} columns={columns} cartControls />
+                        {isPatient && total < 10 ?
```
Comment (vsmalladi) [inline src/encoded/static/components/search.js]: Lets change it to <= 10

**Review by vsmalladi — CHANGES_REQUESTED** [review]
Change to <=10 instead of < 10

File: src/encoded/static/components/search.js (hunk 1 above)
Comment (mingjiecn) [reply src/encoded/static/components/search.js]: Done.

**Review by vsmalladi — APPROVED** [review]

**Comment by vsmalladi** [comment]
LGTM will merge

**Pull request merged**